samples/flexy-pool-sample/build.gradle.kts

```diff
@@ -25,18 +25,9 @@ dependencies {
     implementation("org.apache.commons:commons-io:1.3.2")
 
     testImplementation("org.springframework.boot:spring-boot-starter-test")
-    testImplementation("org.junit.jupiter:junit-jupiter:5.6.2")
 }
 
 tasks {
-    bootRun {
-        val args = args!!
-        if (project.hasProperty("args")) {
-            val userArgs = project.findProperty("args") as String
-            userArgs.split(" ").forEach { args.add(it) }
-        }
-    }
-
```

Comment:
I don't think this should be removed - it allows to pass custom arguments to the application (as in other samples, but without zipkin).

Reply:
Sorry, should have commented on what I was doing here.

I removed it because [Gradle bootRun/JavaExec has `--args` support built in](https://docs.spring.io/spring-boot/docs/2.4.3/gradle-plugin/reference/htmlsingle/#running-your-application-passing-arguments) and wasn't sure why `-P` Gradle property support was required for these.

`./gradlew :samples:datasource-proxy-sample:bootRun --args='--spring.debug=true' --info -Pzipkin=true` ends up in 

`java $CLASSPATH_ETC com.github.gavlyukovskiy.sample.SampleDatasourceProxyApplication --spring.debug=true --spring.zipkin.enabled=true --spring.sleuth.enabled=true`

which is the same as `-Pargs='--spring.debug=true'` using the `-P` support hacked in here.

I think I forgot to remove it from the other bootRun tasks but I think it can be removed there too. I think it'd be better to use the canonical JavaExec support where possible - what do you think? 

As far as I can tell`-Pzipkin` for the other two samples is intended as some convenience semantic sugar so you don't have to remember how to do `bootRun --args='--spring.zipkin.enabled=true --spring.sleuth.enabled=true'` which is a bit of a mouthful. But if you want to add extra args you can do it the normal `bootRun`/`javaExec` way.

Reply:
I think back then when I wrote it Gradle didn't support `args` so I came up with my own :) Nice to know that things are changed. I agree it's better to use it and it can be removed from other two samples

> As far as I can tell-Pzipkin for the other two samples is intended as some convenience semantic sugar

Yes that was exactly the reason. Now i'm thinking about it and maybe it's better to use profiles for that:
```
./gradlew :samples:datasource-proxy-sample:bootRun --args='--spring.profiles.active=zipkin'
```
it's far more understandable for someone who's trying to get everything work without looking how my Gradle script interprets `-Pzipkin`

Reply:
Good idea! Added onto this PR [here](https://github.com/gavlyukovskiy/spring-boot-data-source-decorator/pull/57/commits/3110de756eafe4cf3fdb59766137c20291d090c6).